pkg/controller/scansettingbinding/scansettingbinding_controller.go

```diff
@@ -465,9 +465,16 @@ func profileReferenceToScan(reference *profileReference) (*compliancev1alpha1.Co
 		Name:               reference.name,
 	}
 
-	err = fillContentData(reference.profileBundle, &scan)
-	if err != nil {
-		return nil, "", err
+	hasCustomRuleTP := false
+	if reference.tailoredProfile != nil {
+		hasCustomRuleTP = hasCustomRule(reference.tailoredProfile)
+	}
+
+	if !hasCustomRuleTP {
+		err = fillContentData(reference.profileBundle, &scan)
```

Comment:
This means tailored profiles can only contain CEL rules or SCAP rules right? If we detect any CEL rules in the tailored profiles, we'll short-circuit this step. Do we have a test case for that?

That also means we can't have a mix of CEL and SCAP rules in the same profile?

Reply:
No, We can't mix CEL and SCAP rules in the same tailored profile